build.gradle.kts

```diff
@@ -55,7 +55,3 @@ fun <T> Project.whenEvaluated(action: Project.() -> T) {
         afterEvaluate { action() }
     }
 }
-
-rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
-    rootProject.the<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension>().nodeVersion = "16.0.0"
-}
```

Comment:
Node.js 16.x.x should be default for Kotlin 1.6.20, so this workaround should no longer be needed.